src/queries/page.server.ts

```diff
@@ -69,11 +85,320 @@ export const fetchGetPagesBySite = async (
   })
 }
 
+const getImageDimensionByUri = async (
```

Comment:
There is already an API to get the size of the image https://github.com/Crossbell-Box/xLog/blob/dev/src/app/api/image/route.ts